_posts/2022-12-05-solidity-core-team-updates.md

```diff
@@ -0,0 +1,142 @@
+
+#### What would you like to see in the Solidity ecosystem in the future?
+
+_Solidity will start having a more community-friendly roadmap and I would love it if the community could engage with us on those issues, especially in our forum since they can positively influence our plans. Another personal motivation is improving developer ergonomics with better debugging tools or documentation._
+
+#### What do you like to do in your free time?
+
+📺 _Besides being a professional couch potato and a sucker for good movies and TV shows, I also occasionally do some light gaming._
+🤖 _When I have the right inspiration I also work on my IoT projects in my house where I have to negotiate with the most demanding stakeholder ever, my wife._
+⛳️ _I am also an enthusiastic amateur golfer, which means on a sunny, windless day I'm probably in the links doing some shots!_
+
+#### What made you join the Solidity team?
+
+📣 _The culture on this team is unlike anything I have ever experienced! We are the embodiment of radical candor by caring about the people we talk to and by not being afraid of saying what we mean, we do let the best argument win._
+👨‍💻 _As a software engineer myself, helping shape the future of Solidity is very motivational for anyone that has ever had any interest in WEB3._
+
+### Rodrigo - @[r0qs](https://github.com/r0qs)
+
+#### What is your professional background?
+
+*I have a BSc in Computer Science, and after my bachelor's, I worked for almost five years as a system administrator of High-Performance Computing clusters using mainly open-source software and developing all sorts of scripts and tools to manage the systems. Then, I decided that it was time to explore new areas and learn a new programming language, so I did an MSc in Distributed Systems. My MSc thesis focused on implementing and evaluating a Collision-fast Atomic Broadcast protocol based on a variation of the Distributed Consensus problem, which I implemented in Scala.*
+
+*After my master's, I worked as a software engineer in a Brazilian IT company building an API Gateway and API Manager Platform that worked as an abstraction layer for legacy web services. The system was mainly implemented in Javascript, Ruby, and Go.*
+
+*Over the years, I developed some small projects in Solidity in my free time, and eventually, in 2018, I decided to start a PhD in Distributed Systems to fulfill my desire to learn more about Ethereum. My PhD was focused on designing and developing a transparent and privacy-preserving verifiable credential system for academic diplomas that could work on a global scale. The core functionality was implemented in Solidity and Circom. Finally, in 2022, I had the opportunity to join the Solidity team.*
```

Comment:
```suggestion
*I have a BSc in Computer Science, and after my bachelor's, I worked for almost five years as a system administrator of High-Performance Computing clusters using mainly open-source software and developing all sorts of scripts and tools to manage the systems. Then, I decided that it was time to explore new areas and learn a new programming language, so I did an MSc in Distributed Systems. My MSc thesis focused on implementing and evaluating a Collision-fast Atomic Broadcast protocol based on a variation of the Distributed Consensus problem, which I implemented in Scala.*

*After my master's, I worked as a software engineer in a Brazilian IT company building an API Gateway and API Manager Platform that worked as an abstraction layer for legacy web services. The system was mainly implemented in Javascript, Ruby, and Go.*

*Over the years, I developed some small projects in Solidity in my free time, and eventually, in 2018, I decided to start a Ph.D. in Distributed Systems to fulfill my desire to learn more about Ethereum. My Ph.D. was focused on designing and developing a transparent and privacy-preserving verifiable credential system for academic diplomas that could work on a global scale. The core functionality was implemented in Solidity and Circom. Finally, in 2022, I had the opportunity to join the Solidity team.*
```

Reply:
I think ``PhD``and ``Ph.D.`` are both accepted nowadays and we also didn't use ``.`` in MSc or BSc so would go without ``.`` here as well for consistency.